python/app/qt_importer.py

```diff
@@ -17,16 +17,38 @@
 try:
     from sgtk.platform.qt import QtCore, QtGui
 except ImportError:
+    __PySide_QtGui = None
+    __PySide_QtWidgets = None
+    imported_qt = False
     try:
-        # Try to import from PySide2 and keep the QtGui interface
-        # consistent with PySide (Qt 4).
-        # We don't try to cover all cases, only the QtGui module as we
-        # are using components from that which are different between Qt 5 and 4.
-        # Approach is taken from the `tk-core` `pyside2_patcher.py` module.
+        # Try to import from PySide2
```

Comment:
Yes, a user could have PySide2 and PySide6 installed in their Python environment at the same time.

Here, we just want to find a version of PySide in the user's environment that we can use to run the console, and we give PySide2 precedence since this version has been in use for some time and should be more stable than the newer PySide6.